src/app/common/h/service/AgentAuthService.ts

```diff
@@ -10,7 +10,7 @@ export class AgentAuthService {
 
   login(username: string, password: string) {
     return this.http.post(
-      "http://127.0.0.1:8080/login",
+      "http://3.135.107.110:8080/login",
```

Comment:
This looks like it will cause trouble. We should use variables from `environment` for all URLs.